### PR TITLE
Add KUBE documentation to the website

### PR DESCRIPTION
**Pull request by rgrandl**

### CODE REVIEW

File: website/docs.md [hunk 1]
```diff
@@ -1986,6 +1986,350 @@ with the same arg value`.
...
+```go
+// ./examples/customkube
+...
+
+const jaegerPort = 14268 // Port on which the Jaeger service is receiving traces.
+
+func main() {
+  // Invokes the `KUBE` deployer with the plugin to export traces as instructed
+  // by handleTraceSpans.
+  tool.Run("deploy", tool.Plugins{
```
Comment (mwhittaker) [inline website/docs.md:2267]: This call to `tool.Run` has to be below the definition of `handleTraceSpans`. I also realized that I used the name `"deploy"` here, but that was a bad name. It should be `"customkube"` in this example.

**Review by mwhittaker — APPROVED** [review]
Nice!

File: website/docs.md [hunk 2]
```diff
@@ -1986,6 +1986,350 @@ with the same arg value`.
...
+* You can use existing tools to deploy your application (e.g., [kubectl][kubectl],
+CI/CD pipelines like [Github Actions][githubactions], [Argo CD][argocd] or
+[Jenkins][jenkins]).
+
+## Overview
+
+Figure below shows the high level overview of the `KUBE` deployer. The user
+provides the application binary and a configuration file `config.yaml`. The deployer builds a
+container image for the application, and generates Kubernetes resources that
+enables the application to run in a Kubernetes cluster `deployment.yaml`.
```
[the remark this answers is not in the document]
Comment (rgrandl) [reply website/docs.md]: Done

File: website/docs.md [hunk 3]
```diff
@@ -1986,6 +1986,350 @@ with the same arg value`.
...
+
+Note that the generated Kubernetes resources encapsulate information provided by
+the user in the `config.yaml`. For example, the user can colocate components into
+groups ([`Foo`, `Bar`]), specify resource requirements for running pods, min and
+max replicas, mount volumes, etc. More details on configuration options [here](#kube-config).
+
+By default, the `KUBE` deployer exports logs to `stdout` and discards metrics
+and traces. To customize how to export telemetry data, you have to use the
+`KUBE` [plugin API][kube_telemetry_api] to register plugins that contains
+implementations on how to export logs, metrics and traces. [Here][kube_telemetry]
```
[the remark this answers is not in the document]
Comment (rgrandl) [reply website/docs.md]: Done

File: website/docs.md [hunk 4]
```diff
@@ -1986,6 +1986,350 @@ with the same arg value`.
...
+is an example how to export metrics to [Prometheus][prometheus] and traces to
```
[the remark this answers is not in the document]
Comment (rgrandl) [reply website/docs.md]: Done

File: website/docs.md [hunk 5]
```diff
@@ -1986,6 +1986,350 @@ with the same arg value`.
...
+```yaml
+appConfig: weaver.toml
+repo: docker.io/mydockerid
+
+listeners:
+  - name: hello
+    public: true
+```
+
+The `KUBE` configuration file contains a reference to the application config
```
[the remark this answers is not in the document]
Comment (rgrandl) [reply website/docs.md]: Done

File: website/docs.md [hunk 6]
```diff
@@ -1986,6 +1986,350 @@ with the same arg value`.
...
+|----------------|-----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| appConfig      | required  | Path to the Service Weaver application config file.                                                                                                                                                                                                                                      |
+| image          | optional  | Name of the container image hosting the Service Weaver application. If absent, the image name defaults to `<app_name>:<app_version>`.                                                                                                                                                    |
```
[the remark this answers is not in the document]
Comment (rgrandl) [reply website/docs.md]: Done

File: website/docs.md (hunk 1 above)
Comment (rgrandl) [reply website/docs.md]: Done

File: website/docs.md [hunk 7]
```diff
@@ -1986,6 +1986,350 @@ with the same arg value`.
...
+...
+Uploading image to docker.io/mydockerid/...
+...
+Generating kube deployment info ...
+...
+kube deployment information successfully generated
+/tmp/kube_ffa65856.yaml
+```
+
+`/tmp/kube_ffa65856.yaml` contains the generated Kubernetes resources for the
```
[the remark this answers is not in the document]
Comment (rgrandl) [reply website/docs.md:2097]: Done

**Review by rgrandl — COMMENTED** [review]
Thanks Michael!

**Pull request merged**